src/iidm/NodeBreakerVoltageLevel.hpp

```diff
@@ -105,8 +114,18 @@ class NodeBreakerVoltageLevel : public VoltageLevel {
 
     unsigned long getNode2(const std::string& switchId) const;
 
+    /**
+     * @deprecated Use {@link #getMaximumNodeIndex()} instead.
+     */
     unsigned long getNodeCount() const;
```

Comment:
If the method is private and deprecated, why do you keep it?

Reply:
Ok removed